cluster-operator/src/test/java/io/strimzi/operator/cluster/ClusterOperatorConfigTest.java

```diff
@@ -32,23 +32,24 @@
 
 public class ClusterOperatorConfigTest {
 
-    private static Map<String, String> envVars = new HashMap<>(8);
+    private static final Map<String, String> ENV_VARS = new HashMap<>(8);
```

Comment:
Wouldn't keeping this for a separate PR make the review much easier?

Reply:
I guess so. I have a habit of making things `final` that can be final, but this then invoked the Checkstyle `ConstantName` rules. I'll remove the `final`.

Reply:
I normally note things like this and open a small separate PR for them later.

Reply:
Will do :)